python/lsst/ctrl/stats/data/dbEntry.py

```diff
@@ -33,15 +32,33 @@ def __init__(self, dbList):
         """
         # the DAG node name
         self.dagNode = dbList[0]
+
         # the host where the job resided
         self.executionHost = dbList[1]
+
         # the name of the slot
         self.slotName = dbList[2]
+
         # the time the job was submitted to HTCondor
-        self.submitTime = dbList[3]
+        if dbList[3] == 0:
+            self.submitTime = None
+        else:
+            self.submitTime = dbList[3]
+
```

Comment:
I'd like to keep these how they are now.  I realize that it's acceptable python to write things that way, and while it's a little verbose, I find it more readable.